nodejs/versions.json

```diff
@@ -0,0 +1 @@
+{"21.7.2":"b4b1e2a07e96f85f6ce34a2fbfea348691aefe5cb219aa6951e23ccc991f9e2f","21.7.1":"1272b6e129d564dbde17527b844210b971c20a70ae729268186b7cb9d990a64b","21.7.0":"e41eefe1e59624ee7f312c38f8f7dfc11595641acb2293d21176f03d2763e9d4","21.6.2":"191294d445d1e6800359acc8174529b1e18e102147dc5f596030d3dce96931e5","21.6.1":"7a82f356d1dcba5d766f0e1d4c750e2e18d6290b710b7d19a8725241e7af1f60","21.6.0":"20265bfcfa73c8b46b32378641d38b009dfc980eb28192c3d5ab7f6986fdb1e3","21.5.0":"afd7d4713573cd814f7e4df320de8d5c8e147b4101bc9fbbe2a6d52eb5f8b072","21.4.0":"7a80f6527654602d7358c5be2eefc4f80a64c8901630a83977b073c34f25479c","21.3.0":"ab4172ec827f770c6c3b4b6f30824104137eda474848e84d55ed55b341e67725","21.2.0":"d57c9cea394764fa1d9af51e52c7449f71193e9d44c4a81fbedec653ec827707","21.1.0":"91ac72e4444c5e5ab4b448030a61ffa95acd35d34a9d31d2d220ee2bed01b925","21.0.0":"bc56192b951ad183506dca9acf7a4d0c02591140b7fc8f25661375199266f3f2"}
```

Comment:
Can we consider putting each version on its own line? I think that could reduce the diffs substantially.

Reply:
ooh yeah i didn't think about that

Reply:
`gen.ts` now writes the file in a way that satisfies biome :)